cmd/client-s3.go

```diff
@@ -54,6 +56,19 @@ const (
 	googleHostName = "storage.googleapis.com"
 )
 
+// getNumMultiPartThreads gets the number of multipart threads to be used for mc cp/mirror functions
+func getNumMultiPartThreads() uint {
+	multipartThreadStr := os.Getenv("DEFAULT_MULTIPART_THREADS")
+	if multipartThreadStr != "" {
+		numThreads, err := strconv.ParseUint(multipartThreadStr, 10, 32)
+		if err != nil {
```

Comment:
We should also perhaps restrict users from not setting this value too high i.e not beyond their runtime.NCPU() range.. also the env should have MC_ prefix. 

MC_THREADS or MC_MULTIPART_THREADS

Reply:
done.

Reply:
ncpu() is not exposed by runtime package. NumCPU() isn't relevant. Could you please clarify?